Modules/IO/TransformMINC/src/itkMINCTransformIO.cxx

```diff
@@ -31,15 +31,20 @@
 #include "itkVector.h"
 #include "itkDisplacementFieldTransform.h"
 #include "itkMetaDataObject.h"
+#include "itkImageRegionIterator.h"
+#include "vnl/vnl_vector_fixed.h"
+
+#include "itkMINCImageIOConfigurePrivate.h"
+
 
 namespace itk
 {
 
 template <typename TParametersValueType>
 MINCTransformIOTemplate<TParametersValueType>::MINCTransformIOTemplate()
-{
-  m_XFM_initialized = false;
-}
+  : m_XFM_initialized(false)
+  , m_RAStoLPS(ITK_MINC_IO_RAS_TO_LPS)
```

Comment:
Initialize these in the header file and remove the (empty) constructor implementation.